src/chrome/content/rules/GemCutter.org.xml

```diff
@@ -0,0 +1,13 @@
+<!--
+	Invalid certificate:
+		gemcutter.org
+
+-->
+<ruleset name="GemCutter.org">
+
+	<target host="gemcutter.org" />
```

Comment:
Same for `www`.
The file should be referenced in `RubyGems.xml`.

Reply:
When you go through the invalid certificate warning for `www`:

> Fastly error: unknown domain: www.gemcutter.org. Please check that this domain has been added to a service.